ci-operator/step-registry/cucushift/installer/check/ibmcloud/private/cucushift-installer-check-ibmcloud-private-commands.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# IBM Cloud CLI login
+function ibmcloud_login {
+    export IBMCLOUD_CLI=ibmcloud
+    export IBMCLOUD_HOME=/output
+    region="${LEASED_RESOURCE}"
+    rg=$1
+    export region
+    echo "Try to login to ${rg}..."
+    "${IBMCLOUD_CLI}" login -r ${region} -g ${rg} --apikey @"${CLUSTER_PROFILE_DIR}/ibmcloud-api-key"
+}
+
+INSTALL_CONFIG="${SHARED_DIR}/install-config.yaml"
+INFRA_ID=$(jq -r .infraID ${SHARED_DIR}/metadata.json)
+RESOURCE_GROUP=$(yq-go r "${INSTALL_CONFIG}" 'platform.ibmcloud.resourceGroupName')
+if [[ -z "${RESOURCE_GROUP}" ]]; then
+    RESOURCE_GROUP="${INFRA_ID}-rg"
```

Comment:
In imbcloud, the resource group created by installer, would also append `-rg` automatically?